download-data.py

```diff
@@ -54,35 +367,18 @@ def main() -> None:
         " To switch back, rerun this script with the `--release current` option.",
     )
     parser.add_argument(
-        "--format",
+        "--module-results",
+        "--modules",
         type=str,
-        help=(
-            "The format to download the data in. Either 'SCE' or 'AnnData'."
-            " Defaults to 'SCE'."
-            " For more than one format, use a comma separated list with no spaces."
-        ),
-        default="SCE",
-    )
-    parser.add_argument(
-        "--include",
-        type=str,
-        default="processed",
-        help=(
-            "The level of processing for the experiment files to include."
-            " One or more of 'processed', 'filtered', 'unfiltered', or 'bulk'."
-            " Defaults to 'processed'."
-            " For more than one level, use a comma separated list with no spaces."
-        ),
-    )
-    parser.add_argument(
-        "--include-reports",
-        action="store_true",
-        help="Include html report files in the download.",
+        default="",
+        help="The results modules to download."
+        " A comma separated list of results modules to download."
+        "\nWhen this option is selected, the `--process-stage` and `--include-reports` options are ignored.",
```

Comment:
My understanding is that if these are used then you only download the results and not release, so I would specify that in this help message.